webapp/components/DonationInfo/DonationInfo.spec.js

```diff
@@ -37,7 +37,7 @@ describe('DonationInfo.vue', () => {
     ).toBe('donations.donate-now')
   })
 
-  it('creates a title from the current month and a translation string', () => {
+  it.skip('creates a title from the current month and a translation string', () => {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-10T18:50:34Z" title="Friday, January 10th 2020, 7:50:34 pm +01:00">Jan 10, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -37,7 +37,7 @@ describe('DonationInfo.vue', () => {
     ).toBe('donations.donate-now')
   })
 
-  it('creates a title from the current month and a translation string', () => {
+  it.skip('creates a title from the current month and a translation string', () => {
```


Hey @mattwr18 can you tell what's the matter with the tests?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-10T19:01:51Z" title="Friday, January 10th 2020, 8:01:51 pm +01:00">Jan 10, 2020</time>_

---


@roschaefer I added it in my commit message. These tests are testing functionality that is no longer used in production. It was failing the build, and instead of refactoring, I've left the tests, but skipped them till we decide whether to remove completely or reuse the donations functionality